signing/verifier.go

```diff
@@ -0,0 +1,32 @@
+package signing
+
+import (
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/spacemeshos/ed25519"
+)
+
+type Verifier interface {
```

Comment:
Why do you need to define the interface here, instead of where you use the interface like the go convention?

Reply:
Fixed